remoteworkitem/scheduler.go

```diff
@@ -67,7 +67,7 @@ func (s *Scheduler) ScheduleAllQueries(ctx context.Context, accessTokens map[str
 						return errors.WithStack(err)
 					}
 					// Convert the remote item into a local work item and persist in the DB.
-					_, err = ConvertToWorkItemModel(ctx, tx, tq.TrackerID, i, tq.TrackerType, tq.SpaceID)
+					_, err = ConvertToWorkItemModel(ctx, tx, tq.TrackerID, i, tq.TrackerType, tq.SpaceID, uuid.NewV4())
```

Comment:
Why do you need to change function signature of `ConvertToWorkItemModel(...)`? 

You could just pass 'uuid.NewV4()' to workitem Save function directly -> https://github.com/fabric8-services/fabric8-wit/pull/2176/files#diff-1afcce7c6090c7de7652ad1efd132cabR158

Reply:
The call stack goes like this

`ConvertToWorkItemModel` -> `upsert` -> `wir.Save` that is why the function signature needs to change.